.github/workflows/test-schedulers.yaml

```
@@ -0,0 +1,24 @@
+name: Test Schedulers
```

Just to be in sync with the rest of workflow titles.

```suggestion
name: ReFrame CI / Scheduler backend tests
```